CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # OpenKit Native ( C/C++ ) Changelog
 
+## [Unreleased](https://github.com/Dynatrace/openkit-native/compare/v1.0.0...HEAD)
```

Comment:
We have one bullet point in the "Changed" list and 69 commits in the compare link.